contracts/eosio.msig/README.md

```diff
@@ -46,7 +46,7 @@ Cleos usage example.
 
 Prerequisites:
    - eosio.token contract installed to eosio.token account, eosio.msig contract installed on eosio.msig account which is a privileged account.
-   - account 'treasury' is the issuer of EOS token.
+   - account 'treasury' is the issuer of EOS token.Make sure that treasury has the "eosio.msig@eosio.code" permission.
```

Comment:
This should not be necessary if the prerequisite above "eosio.msig account which **is a privileged account**." is fulfilled (emphasis mine). 

If that is not the case this is a bug in the code and not a bug in the tutorial